bridge/slack/handlers.go

```diff
@@ -132,7 +137,12 @@ func (b *Bslack) skipMessageEvent(ev *slack.MessageEvent) bool {
 		ev.SubMessage.Edited == nil {
 		return true
 	}
-	return b.filesCached(ev.Files)
+
+	if len(ev.Files) > 0 {
```

Comment:
Could you please create a separate PR for this fix? That will enable it to be merged ASAP rather than having to wait on all outstanding discussions to concluded. Also: this fix is entirely unrelated to this PR.